frontend/src/components/modals/EditResponseModal.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import { Modal, Button } from 'react-bootstrap'
+import { useTranslation } from 'react-i18next/'
+import { useHistory } from 'react-router-dom'
+
+const EditResponseModal = ({
+  setShowResponseModal,
+  showResponseModal,
+  title,
+}) => {
+  const { t } = useTranslation()
+  let history = useHistory()
+
+  const handleClick = () => {
+    setShowResponseModal(false)
+    if (!title.includes('unsuccessful')) {
```

Comment:
Szerintem érzed, hogy ez nem túl elegáns megoldás. Egyrészt egy stringben keresni egy másikat és ez alapján dönteni el, hogy sikerült vagy sem... Másrészt lehet nem a modal-nak a felelősségi köre navigálni, szerintem neki egy onClose eseménye lehetne, ha bezárják és majd kint navigálunk.

Reply:
Először egyel feljebb a try-catch-be került, azonban ott a Modal nem jelent meg, mivel a history.push is azonnal lefutott és átnavigált egy másik oldalra. Igen, nem volt elegáns. :-1: Hm, többet és előre írnom kellene neked, ha nem vagyok megelégedve a megoldással.